cmd/upgrade/operatorroles/cmd.go

```diff
@@ -340,108 +291,26 @@ func upgradeOperatorPolicies(mode string, r *rosa.Runtime,
 	return nil
 }
 
-func createOperatorRole(
-	mode string, r *rosa.Runtime, cluster *cmv1.Cluster, prefix string, missingRoles map[string]*cmv1.STSOperator,
-	policies map[string]*cmv1.AWSSTSPolicy, unifiedPath string, managedPolicies bool) error {
-	accountID := r.Creator.AccountID
-	switch mode {
-	case aws.ModeAuto:
-		err := upgradeMissingOperatorRole(missingRoles, cluster, accountID, prefix, r,
-			policies, unifiedPath, managedPolicies)
-		if err != nil {
-			return err
-		}
-		helper.DisplaySpinnerWithDelay(r.Reporter, "Waiting for operator roles to reconcile", 5*time.Second)
-	case aws.ModeManual:
-		commands, err := roles.BuildMissingOperatorRoleCommand(
-			missingRoles, cluster, accountID, r, policies, unifiedPath, prefix, managedPolicies)
-		if err != nil {
-			return err
-		}
-		if r.Reporter.IsTerminal() {
-			r.Reporter.Infof("Run the following commands to create the operator roles:\n")
-		}
-		fmt.Println(commands)
-	default:
-		r.Reporter.Errorf("Invalid mode. Allowed values are %s", aws.Modes)
-		os.Exit(1)
+func handleModeFlag(cmd *cobra.Command, mode string) (string, error) {
```

Comment:
Is this the same function from upgrade roles?

Reply:
Functions are similar, but not the same.

Reply:
Removed the `handleModeFlag` function from `upgrade roles` since we're going to call it once.
Now it is like it was previously.